doc/deploy_starrocks_with_operator_howto.md

```diff
@@ -0,0 +1,297 @@
+
+    ```YAML
+    spec:
+      starRocksFeSpec:
+        service:            
+          type: LoadBalancer # specified as LoadBalancer
+    ```
+
+2. Obtain the IP address `EXTERNAL-IP` and port `PORT(S)` that the FE Service exposes to the outside.
+
+    ```Bash
+    $ kubectl -n starrocks get svc
+    NAME                                 TYPE           CLUSTER-IP       EXTERNAL-IP                                                              PORT(S)                                                       AGE
+    starrockscluster-sample-be-search    ClusterIP      None           <none>        9050/TCP                                                      6m39s
+    starrockscluster-sample-be-service   ClusterIP      10.96.86.207   <none>        9060/TCP,8040/TCP,9050/TCP,8060/TCP                           6m39s
+    starrockscluster-sample-fe-search    ClusterIP      None           <none>        9030/TCP                                                      8m
+    starrockscluster-sample-fe-service   LoadBalancer   10.96.26.146   a7509284bf3784983a596c6eec7fc212-618xxxxxx.us-west-2.elb.amazonaws.com     8030:30028/TCP,9020:32241/TCP,9030:32640/TCP,9010:32384/TCP   8m
+    ```
+
+3. Log in to your machine host and access the StarRocks cluster by using the MySQL client.
+
+    ```Bash
+    mysql -h a7509284bf3784983a596c6eec7fc212-618xxxxxx.us-west-2.elb.amazonaws.com -P9030 -uroot
+    ```
+
+### 3.2. Upgrade StarRocks Cluster
+
+#### 3.2.1. Upgrade BE nodes
+
+Run the following command to specify a new BE image file, such as `starrocks/be-ubuntu:latest`:
+
+```bash
+kubectl -n starrocks patch starrockscluster starrockscluster-sample --type='merge' -p '{"spec":{"starRocksBeSpec":{"image":"starrocks/be-ubuntu:latest"}}}'
+```
+
+#### 3.2.2. Upgrade FE nodes
+
+Run the following command to specify a new FE image file, such as `starrocks/fe-ubuntu:latest`:
+
+```bash
+kubectl -n starrocks patch starrockscluster starrockscluster-sample --type='merge' -p '{"spec":{"starRocksFeSpec":{"image":"starrocks/fe-ubuntu:latest"}}}'
+```
+
+The upgrade process lasts for a while. You can run the command `kubectl -n starrocks get pods` to view the upgrade
+progress.
+
+### 3.3. Scale StarRocks cluster
+
+This topic takes scaling out the BE and FE clusters as examples.
+
+#### 3.3.1. Scale out BE cluster
+
+Run the following command to scale out the BE cluster to 9 nodes:
+
+```bash
+kubectl -n starrocks patch starrockscluster starrockscluster-sample --type='merge' -p '{"spec":{"starRocksBeSpec":{"replicas":9}}}'
+```
+
+#### 3.3.2. Scale out FE cluster
+
+Run the following command to scale out the FE cluster to 4 nodes:
+
+```bash
+kubectl -n starrocks patch starrockscluster starrockscluster-sample --type='merge' -p '{"spec":{"starRocksFeSpec":{"replicas":4}}}'
+```
+
+The scaling process lasts for a while. You can use the command `kubectl -n starrocks get pods` to view the scaling
+progress.
+
+### 3.4. Using ConfigMap to configure your StarRocks cluster
```

Comment:
Add cautions on scale-in FE nodes, 1. can only scale-in step by step, and each time less than half of the nodes can be scaled-in.

2. can't do 3->1 scale in.